SIS_import/sis_set_url.rb

```diff
@@ -549,84 +549,22 @@ def update_MPathway_with_Canvas_url(outputDirectory)
-					if (env_array.size != 6)
-						return_hash["error"] = "security file should have the settings in format of: Canvas_token=<Canvas token>,Canvas_server=<Canvas server url>,ESB_key=<ESB key>,ESB_secret=<ESB secret>,ESB_URL=<ESB URL>,ESB_TOKEN_URL=<ESB token server URL>"
-						return return_hash
-					end
-					token_array=env_array[0].split('=')
-					@canvasToken=token_array[1]
-					url_array=env_array[1].split('=')
-					@canvasUrl=url_array[1]
-					key_array=env_array[2].split('=')
-					@esbKey=key_array[1]
-					secret_array=env_array[3].split('=')
-					@esbSecret=secret_array[1]
-					url_array=env_array[4].split('=')
-					@esbUrl=url_array[1]
-					token_url_array=env_array[5].split('=')
-					@esbTokenUrl=token_url_array[1]
-					break
-				end
-			end
-		elsif (count==2)
-			if (Dir[arg].length != 1)
-				## token file
-				return_hash["error"] = "Cannot find properties file " + arg
-				return return_hash
-			end
-			File.open(arg, 'r') do |propertiesFile|
-				while line = propertiesFile.gets
-					# only have one line, and in this format:
-					# directory=<current working directory>,page_size=<ESB API call page size>,esb_time_interval=<ESB API call time interval in seconds>,esb_allowed_call_number=<maximum ESB API call number during the interval>,canvas_time_interval=<Canvas API call time interval in secs>,canvas_allowed_call_number=<maximum Canvas API call during the interval>,alert_email_address=ALERT_EMAIL_ADDRESS
-					env_array = line.strip.split(',')
-					if (env_array.size != 7)
-						return_hash["error"] = "Properties file should have the settings in format of: directory=<current working directory>,page_size=<ESB API call page size>,esb_time_interval=<ESB API call time interval in seconds>,esb_allowed_call_number=<maximum ESB API call number during the interval>,canvas_time_interval=<Canvas API call time interval in secs>,canvas_allowed_call_number=<maximum Canvas API call during the interval>,alert_email_address=<alert email address>"
-						return return_hash
-					end
-					diretory_array=env_array[0].split('=')
-					@currentDirectory=diretory_array[1]
-					page_size_array=env_array[1].split('=')
-					@page_size=page_size_array[1]
-					esb_interval_array=env_array[2].split('=')
-					@esb_call_hash["time_interval_in_seconds"]=esb_interval_array[1].to_i
-					esb_call_array=env_array[3].split('=')
-					@esb_call_hash["allowed_call_number_during_interval"]=esb_call_array[1].to_i
-					canvas_interval_array=env_array[4].split('=')
-					@canvas_call_hash["time_interval_in_seconds"]=canvas_interval_array[1].to_i
-					canvas_call_array=env_array[5].split('=')
-					@canvas_call_hash["allowed_call_number_during_interval"]=canvas_call_array[1].to_i
-					alert_email_address_array=env_array[6].split('=')
-					@alert_email_address=alert_email_address_array[1]
-					break
-				end
-			end
-		else
-			# break
-		end
-
-		#increase count
-		count=count+1
-	end
-	return return_hash
+	Dotenv.load
+	@logger.info("ddd")
```

Comment:
What is this for? Should be more descriptive if you want to leave it in here. like "Loading Dotenv"

Reply:
removed!